packages/arb-token-bridge-ui/src/token-bridge-sdk/utils.ts

```diff
@@ -10,3 +11,51 @@ export const getChainIdFromProvider = async (provider: Provider) => {
   const network = await provider.getNetwork()
   return network.chainId
 }
+
+export const getBridgeTransferProperties = async ({
+  sourceChainProvider,
+  destinationChainProvider,
+  sourceChainErc20Address
+}: {
+  sourceChainProvider: Provider
+  destinationChainProvider: Provider
+  sourceChainErc20Address?: string
+}) => {
+  const sourceChainId = await getChainIdFromProvider(sourceChainProvider)
+  const destinationChainId = await getChainIdFromProvider(
+    destinationChainProvider
+  )
+
+  const isParentChainEthereumMainnetOrTestnet =
+    isNetwork(sourceChainId).isEthereumMainnetOrTestnet
+
+  const isParentChainArbitrum = isNetwork(sourceChainId).isArbitrum
+  const isDestinationChainOrbit = isNetwork(destinationChainId).isOrbitChain
+
+  const isDeposit =
+    isParentChainEthereumMainnetOrTestnet ||
+    (isParentChainArbitrum && isDestinationChainOrbit)
+
+  const isNativeCurrencyTransfer =
+    typeof sourceChainErc20Address === 'undefined'
+
+  const isWithdrawal =
+    (isNetwork(sourceChainId).isArbitrum &&
```

Comment:
This is the same as `isParentChainArbitrum` and could be reused. Seems like it's just naming issue.